packages/astro/src/vite-plugin-astro-server/route.ts

```diff
@@ -130,7 +130,7 @@ type HandleRoute = {
 	manifestData: ManifestData;
 	incomingRequest: http.IncomingMessage;
 	incomingResponse: http.ServerResponse;
-	status?: 404 | 500;
+	status?: 404 | 500 | 200;
```

Comment:
Can't it be anything else?

Reply:
The type must match what's returned by `getStatus`, I just updated to make TS happy. I don't think it can be something else because `getStatus` checks our own physical routes

Reply:
```suggestion
	status?: ReturnType<typeof getStatus>;
```